requirements.txt

```diff
@@ -1,5 +1,6 @@
 # Install with "pip install -r requirements.txt"
 
+setuptools==33.1.1
```

Comment:
Please keep this alphabetized. Also consider adding a comment on this setuptools line referencing the issue.

Reply:
I was under the impression that the packages in the list are installed in sequence and to avoid the issue (https://github.com/pypa/setuptools/issues/951) setuptools needs to be at the top. But if thats not the case I will alphabetized.
I will add the comment. According to the issue (FileNotFoundError) as described in https://github.com/pypa/setuptools/issues/951, I need to avoid the setuptools version 35.x.x and onwards. Downgrading of setuptools is a workaround for the issue.

Reply:
Alphabetization not possible here. I verified it.

Reply:
Ah good find -- I didn't realize the order mattered.